Packages/Tracking/Hands/Runtime/Scripts/HandBinder/HandBinder.cs

```diff
@@ -244,9 +244,12 @@ void ScaleFingertips()
                 float ratio = leapFingerLength / fingerTipLength;
                 //Adjust the ratio by an offset value exposed in the inspector and the overal scale that has been calculated
                 float adjustedRatio = (ratio * (finger.fingerTipScaleOffset) - BoundHand.scaleOffset);
-
+                //Adjust the ratio to account for service provider scale, assuming the service provider is uniformly scaled
+                var serviceProviderScale = leapProvider?.gameObject.transform.localScale.x ?? 1f;
```

Comment:
If this scale was taken from `lossyScale` it would also account for Providers that are children of scaled parents

Though not sure if the issue exists in that scenario